packages/charts/src/components/BarChart/BarChart.tsx

```diff
@@ -107,6 +116,7 @@ export interface BarChartProps extends IChartBaseProps {
    * - `width`: bar width, defaults to `auto`
    * - `opacity`: bar opacity, defaults to `1`
    * - `stackId`: bars with the same stackId will be stacked
+   * - `highlightColor`: defines a custom color of a specific element. Overwrites code>color</code> of the element.
```

Comment:
Can you briefly outline the function signature here? Same applies for the `ColumnChart`